redi-connect-front/src/pages/app/me/About.tsx

```diff
@@ -1,27 +1,52 @@
 import React from 'react'
+import { Content } from 'react-bulma-components'
 import FormTextArea from '../../../components/atoms/FormTextArea'
+import FormInput from '../../../components/atoms/FormInput'
 import Editable from '../../../components/molecules/Editable'
 import { RedProfile } from '../../../types/RedProfile'
 import { connect } from 'react-redux'
 import { RootState } from '../../../redux/types'
 
+// import {
+//   menteeCountCapacityOptions
+// } from '../../../config/config'
+
 import {
   profileSaveStart
 } from '../../../redux/user/actions'
 import * as Yup from 'yup'
 
-import { FormikValues, FormikHelpers as FormikActions, useFormik, FormikProps } from 'formik'
+import { FormikValues, useFormik } from 'formik'
+
+// do we really need all these type???
+export type UserType =
```

Comment:
Nope, we don't. People who have access to log in and update their `RedProfile` must "by definition" be approved. So only `mentor` and `mentee` applies.

You can re-use the trick I committed onto your other branch the other way, with `Extends` (I recall you commented it showed you why you dislike TS :))